client/js/components/day/employee.js

```diff
@@ -10,8 +11,8 @@ var Employee = React.createClass({
     return {
       name: '',
       rating: '',
-      gigID: Infinity,
-      employeeID: Infinity
+      gigId: Infinity,
```

Comment:
Just curious about the use of infinity as a default value. I dont have much experience working with infinity in JS so just want to make sure it won't give us odd behavior if we are doing computations based on gigdId that might break if Infinity is the default value. It would be a bad default value if that were the case.